README.md

```diff
@@ -1,119 +1,23 @@
-Idris 2
-=======
+# Idris 2
 
 [![Documentation Status](https://readthedocs.org/projects/idris2/badge/?version=latest)](https://idris2.readthedocs.io/en/latest/?badge=latest)
 [![Build Status](https://github.com/idris-lang/Idris2/actions/workflows/ci-idris2.yml/badge.svg)](https://github.com/idris-lang/Idris2/actions/workflows/ci-idris2.yml)
 
-[Idris 2](https://idris-lang.org/) is a purely functional programming language
-with first class types.
+[Idris 2](https://idris-lang.org/) is a purely functional programming language with first class types.
```

Comment:
This line (and the lines in general) are split at 80 chars to keep plaintext/un-rendered readability. Please restore this  : )

Reply:
ok

Reply:
done (though I may forget as I make more changes)

Reply:
this is curious. CI only complains when lines go over 400 chars. Why so?

Reply:
this is the main remaining task

Reply:
ok i think this is done